packages/react/src/components/HelpText/HelpText.tsx

```diff
@@ -54,29 +53,33 @@ const HelpText = ({
           variant='tertiary'
         >
           <button
-            className={cl(classes.helpTextButton, `fds-focus`, className)}
+            className={cl(
+              `fds-helptext--${size}`,
+              'fds-helptext__button',
```

Comment:
Maybe this should be `__trigger`, so we keep the same name as the react component 🤔

Reply:
hmm, good question 🤔 But trigger is also a button element?

Not sure, on one side, we don't want technical implementations in React dictate css classes... 

I we think purely html+css, I feel `__button` is very clear how and where you use this, `__trigger` might be confusing?

Lets do the others and see how it feels. We can always come back and change names if needed.

Reply:
Agree ✅